symphony/bdk/core/config/model/bdk_authentication_config.py

```diff
@@ -0,0 +1,31 @@
+from symphony.bdk.core.config.model.bdk_rsa_key_config import BdkRsaKeyConfig
+from symphony.bdk.core.config.model.bdk_certificate_config import BdkCertificateConfig
+
+
+class BdkAuthenticationConfig:
+
+    def __init__(self, private_key=None, certificate=None):
+        self._bdk_rsa_key_config = BdkRsaKeyConfig(**private_key) if private_key is not None else BdkRsaKeyConfig()
+        self._bdk_certificate_config = BdkCertificateConfig(**certificate) if certificate is not None else BdkCertificateConfig()
+
+    def is_rsa_authentication_configured(self) -> bool:
+        """Check if the RSA authentication is configured
+
+        :return true if the RSA authentication is configured
+        """
+        return (self.bot_rsa_key_config is not None and self.bot_rsa_key_config.is_configured())
```

Comment:
`self.bot_rsa_key_config is not None` will always be true because of initialization in constructor.